packages/terra-clinical-action-header/package.json

```diff
@@ -23,29 +23,29 @@
   },
   "homepage": "https://github.com/cerner/terra-clinical#readme",
   "devDependencies": {
-    "terra-toolkit": "^0.x"
+    "terra-toolkit": "^0.3.1"
   },
   "peerDependencies": {
-    "react": "15.4.2",
-    "react-dom": "15.4.2",
-    "terra-base": "^0.x",
-    "terra-button": "^0.x",
-    "terra-button-group": "^0.x",
-    "terra-icon": "0.x",
-    "terra-mixins": "^1.0.0",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^0.4.1",
+    "terra-button": "^0.4.1",
+    "terra-button-group": "^0.2.1",
+    "terra-icon": "^0.4.1",
+    "terra-mixins": "^1.1.1",
     "terra-clinical-header": "^0.x",
-    "terra-responsive-element": "^0.x"
+    "terra-responsive-element": "^0.3.1"
```

Comment:
You might already know this, but just a heads up.

"^0.3.1" will not resolve "0.x", 

it will only resolve to "0.3.x"